src/tools/cephfs/JournalTool.cc

```diff
@@ -250,14 +250,36 @@ int JournalTool::main_journal(std::vector<const char*> &argv)
     }
   } else if (command == "reset") {
     bool force = false;
-    if (argv.size() == 2) {
+    if (argv.size() == 1) {
```

Comment:
wait.. shouldn't this throw up help?

Reply:
No, because [std::string command = argv[0]](https://github.com/ceph/ceph/blob/main/src/tools/cephfs/JournalTool.cc#L237). At this point, when `argv.[0] == "reset"`, we've already got the command. So no need of throwing help here. Just issue a warning for adding the option.

Reply:
EDIT: I meant https://github.com/ceph/ceph/blob/main/src/tools/cephfs/JournalTool.cc#L231